src/backend/base/langflow/components/docling/docling_inline.py

```diff
@@ -130,6 +130,58 @@ def _terminate_process_gracefully(self, proc, timeout_terminate: int = 10, timeo
                 self.log("Warning: Process still alive after SIGKILL")
 
     def process_files(self, file_list: list[BaseFileComponent.BaseFile]) -> list[BaseFileComponent.BaseFile]:
+        try:
```

Comment:
I am really not sure whether this should be in this PR... This looks like the docling changes to the file component?